routers/web/auth/auth.go

```diff
@@ -368,9 +368,9 @@ func getUserName(gothUser *goth.User) string {
 func HandleSignOut(ctx *context.Context) {
 	_ = ctx.Session.Flush()
 	_ = ctx.Session.Destroy(ctx.Resp, ctx.Req)
-	ctx.DeleteCookie(setting.CookieUserName)
-	ctx.DeleteCookie(setting.CookieRememberName)
-	middleware.DeleteCSRFCookie(ctx.Resp)
+	ctx.DeleteSiteCookie(setting.CookieUserName)
+	ctx.DeleteSiteCookie(setting.CookieRememberName)
+	ctx.Csrf.DeleteCookie(ctx)
 	middleware.DeleteLocaleCookie(ctx.Resp)
```

Comment:
why do we delete the locale cookie on logout?

we should remove this line as cleanup is done on login too

Reply:
yes it's more an enhancement ... I can submit a one line pull for it too ...